setup/www/resources/config/nodejs.org

```diff
@@ -426,4 +426,6 @@ server {
 
     # Rename 'Stable' release to 'Current'
     rewrite ^/(.*)/download/stable$                               https://$server_name/$1/download/current permanent;
+    
+    rewrite ^/en/docs/guides/debugging_getting_started/           https://$server_name/en/docs/guides/debugging-getting-started/ permanent;
```

Comment:
The guide only exists in those two languages. New translations will use the correct file name. So no regex is needed.